deepseek-model-provider/main.go

```diff
@@ -1,19 +1,27 @@
 package main
 
 import (
-	"fmt"
 	"os"
 
-	"github.com/obot-platform/tools/xai-model-provider/server"
+	"github.com/obot-platform/tools/deepseek-model-provider/server"
+	"github.com/obot-platform/tools/deepseek-model-provider/validate"
 )
 
 func main() {
 	apiKey := os.Getenv("OBOT_DEEPSEEK_MODEL_PROVIDER_API_KEY")
 	if apiKey == "" {
-		fmt.Println("OBOT_DEEPSEEK_MODEL_PROVIDER_API_KEY environment variable not set")
+		validate.PrintError("OBOT_DEEPSEEK_MODEL_PROVIDER_API_KEY environment variable not set")
 		os.Exit(1)
 	}
 
+	args := os.Args[1:]
+	if len(args) == 1 && args[0] == "validate" {
+		if err := validate.Run(apiKey); err != nil {
+			validate.PrintError(err.Error())
+		}
+		os.Exit(0)
```

Comment:
Does `os.Exit(1)` make sense if there is an error?

Reply:
makes sense. added